scene/gui/popup_menu.cpp

```diff
@@ -1003,8 +1003,12 @@ void PopupMenu::set_item_text(int p_idx, const String &p_text) {
 		p_idx += get_item_count();
 	}
 	ERR_FAIL_INDEX(p_idx, items.size());
+	if (items.write[p_idx].text == p_text) {
```

Comment:
You don't need to use `.write[]` since you're only reading, so this would be better:
```suggestion
	if (items[p_idx].text == p_text) {
```